src/sentry/api/endpoints/group_similar_issues.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+import functools
+
+from rest_framework.response import Response
+
+from sentry.api.bases.group import GroupEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Group
+from sentry.similarity import features
+from sentry.utils.functional import apply_values
+
+
+class GroupSimilarIssuesEndpoint(GroupEndpoint):
```

Comment:
This feels so simple that I feel like I must be missing something obvious… :thinking: